components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -1255,11 +1256,8 @@ private void translateROIs(Element imageNode, int image)
       roi.scaleY = parseDouble(roiNode.getAttribute("transScalingY"));
       roi.rotation = parseDouble(roiNode.getAttribute("transRotation"));
       String linewidth = roiNode.getAttribute("linewidth");
-      if (linewidth != null) {
-        try {
-          roi.linewidth = Integer.parseInt(linewidth);
-        }
-        catch (NumberFormatException e) { }
+      if (linewidth != null && !linewidth.trim().isEmpty()) {
+        roi.linewidth = Integer.parseInt(linewidth.trim());
```

Comment:
I think it's possible to have a double - maybe makes sense to use (int) Double.parseDouble and catch NFE just to be safe (since we were doing that anyway).